auto_dev/commands/run.py

```diff
@@ -340,5 +60,50 @@ def run(ctx, agent_public_id: PublicId, verbose: bool, force: bool, fetch: bool)
     logger.info("Agent run complete. 😎")
 
 
+@run.command()
+@click.argument(
+    "service_public_id",
+    type=PublicId.from_str,
+    required=False,
+)
+@click.option("-v", "--verbose", is_flag=True, help="Verbose mode.", default=False)
+@click.option("--force/--no-force", is_flag=True, help="Force overwrite of existing service", default=False)
+@click.option("--fetch/--no-fetch", help="Fetch from registry or use local service package", default=True)
+@click.option("--keysfile", help="Path to the private keys file.", type=click.File(), default="keys.json")
+@click.option("--number_of_agents", "-n", help="Number of agents to run.", type=int, default=1)
+@click.pass_context
+def prod(
+    ctx,
+    service_public_id: PublicId,
+    verbose: bool,
+    force: bool,
+    fetch: bool,
+    keysfile: click.File,
+    number_of_agents: int,
+) -> None:
+    """Run an agent in production mode.
+
+    Example usage:
+        adev run prod eightballer/my_service
+    """
```

Comment:
can we include the params in this docstring - the docs will fail if not (can also do later lol)